config/advisory_templates.yml

```diff
@@ -247,57 +247,45 @@ boilerplates:
       topic: |
-        Red Hat build of MicroShift release 4.{MINOR}.{PATCH} is now available with updates to packages and images that fix several bugs. 
-
-        This release includes a security update for the Red Hat build of MicroShift 4.{MINOR}. 
-
-        Red Hat Product Security has rated this update as having a security impact of (impact). A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.
+        Red Hat build of MicroShift release 4.{MINOR}.{PATCH} is now available with updates to packages and images that include a security update.
+        
+        Red Hat Product Security has rated this update as having a security impact of Moderate. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.
       description: |
-        Red Hat build of MicroShift is Red Hat's light-weight Kubernetes orchestration solution designed for edge device deployments and is built from the edge capabilities of Red Hat OpenShift. MicroShift is an application that is deployed on top of Red Hat Enterprise Linux devices at the edge, providing an efficient way to operate single-node clusters in these low-resource environments.
-
-        This advisory contains the RPM packages for Red Hat build of MicroShift 4.{MINOR}.{PATCH}. See the following advisory for the container images for this release:
-
-        https://access.redhat.com/errata/RH[X]A-[YYYY:1234]
-
-        Security Fix(es):
-
-        * [component-1: Description of issue and fix #1 (CVE-YYYY-NNNN)]
-
-        * [component-2: Description of issue and fix #2 (CVE-YYYY-NNNN)]
-
-        * [component-3: Description of issue and fix #3 (CVE-YYYY-NNNN)]
-
-        For more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section. All of the bug fixes may not be documented in this advisory. See the following release notes documentation for details about these changes:
-
-        https://access.redhat.com/documentation/en-us/microshift/4.{MINOR}/html/release_notes/index
+        Red Hat build of MicroShift is Red Hat's light-weight Kubernetes orchestration solution designed for edge device deployments and is built from the edge capabilities of Red Hat OpenShift Container Platform. MicroShift is an application that is deployed on top of Red Hat Enterprise Linux devices at the edge, providing an efficient way to operate single-node clusters in these low-resource environments.
 
+        This advisory contains the RPM packages for Red Hat build of MicroShift 4.{MINOR}.{PATCH}. Read the following advisory for the container images for this release:
+        
+        https://access.redhat.com/errata/RHSA-2024:XXXX
```

Comment:
```suggestion
        https://access.redhat.com/errata/RHSA-XXXX:XXXX
```
do we have an attribute for the year?

Reply:
I'm planning on working on a follow up PR for our automation to "update" the Docs text with the advisory URL. For now, I was thinking about focusing on the text thats supplied when we are requesting to create a new advisory

Reply:
i've applied your suggestion to the PR